modules/articles/client/config/articles.client.routes.js

```diff
@@ -38,10 +46,13 @@
         controller: 'ArticlesController',
         controllerAs: 'vm',
         resolve: {
-          articleResolve: getArticle
+          articleResolve: getArticle,
+          pageTitle: ['articleResolve', function (articleResolve) {
+            return 'MEAN.JS - Edit Article ' + articleResolve.title;
```

Comment:
Does this have to have it's own resolve? Can the `pageTitle` be updated when the `getArticle` promise is resolved?

I think Andrew mentioned that in his comment.. something like..

```javascript
function getArticle($stateParams, ArticleService) {
  var articlePromise = ArticleService.get({
    articleId: $stateParams.articleId
  }).$promise;

  articlePromise.then(function (article) {
    this.data.pageTitle += article.title;
  });

  return articlePromise;
}
```

@trainerbill is that along the lines of what you were thinking? And would that work?